examples/animation/animate_decay.py

```diff
@@ -50,5 +50,7 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, interval=100, init_func=init)
+# Only save last 100 frames, but run forever
+ani = animation.FuncAnimation(fig, run, data_gen, interval=100, init_func=init,
+                              save_count=100)
```

Comment:
Is this needed for sphinx-gallery? There isn't otherwise any saving going on in any of these examples.

Reply:
yeah, SG saves both a gif thumbnail and a js_html output to embed in the docs.